composer.json

```diff
@@ -20,7 +20,8 @@
         }
     ],
     "require" : {
-        "php": ">=7.4"
+        "php": ">=8.0",
+        "drupal/hal-hal": "^1.0"
```

Comment:
For D10, this seems it should be `"drupal/hal": "^2"` at least per https://www.drupal.org/project/hal